### ISSUE TITLE
Cannot install labelx 2.3.0

### ISSUE THREAD

**Issue by saily** [dalwar23/labelx#13]
* labelx version: 2.3.0
* Python version: 3.8.10
* Operating System: MacOS

### Description

Want to install or upgrade to new `labelx >= 2.3.0`. 

### What I Did

```
pip install labelx>=2.3.0
...
Collecting labelx>=2.3.0
  Downloading labelx-2.3.0-py2.py3-none-any.whl (14 kB)
ERROR: Could not find a version that satisfies the requirement Click>=8.04 (from labelx) (from versions: 0.1, 0.2, 0.3, 0.4, 0.5, 0.5.1, 0.6, 0.7, 1.0, 1.1, 2.0, 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 3.0, 3.1, 3.2, 3.3, 4.0, 4.1, 5.0, 5.1, 6.0, 6.1, 6.2, 6.3, 6.4, 6.5, 6.6, 6.7.dev0, 6.7, 7.0, 7.1, 7.1.1, 7.1.2, 8.0.0a1, 8.0.0rc1, 8.0.0, 8.0.1, 8.0.2, 8.0.3, 8.0.4, 8.1.0)
ERROR: No matching distribution found for Click>=8.04
```

Typo on https://github.com/dalwar23/labelx/blob/dev/setup.py#L13, it should be `Click >= 8.0.4`.

**Comment by dalwar23** [comment]
I have addressed the issue and released `v2.3.1`.

**Comment by saily** [comment]
@dalwar23 great - thank you!
